etc/scripts/install-on-okd.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+function enable_admission_webhooks {
+
+  echo "API_SERVER=$API_SERVER"
+  echo "KUBE_SSH_USER=$KUBE_SSH_USER"
+  echo "KUBE_SSH_KEY=$KUBE_SSH_KEY"
+  
+  ssh $KUBE_SSH_USER@$API_SERVER -i $KUBE_SSH_KEY /bin/bash << "EOF"
+  sudo su - root
+  cp -n /etc/origin/master/master-config.yaml /etc/origin/master/master-config.yaml.backup
+  cp -p /etc/origin/master/master-config.yaml /etc/origin/master/master-config.yaml.prepatch
+  cat > /etc/origin/master/master-config.patch << EOT
+admissionConfig:
+  pluginConfig:
+    MutatingAdmissionWebhook:
+      configuration:
+        apiVersion: apiserver.config.k8s.io/v1alpha1
+        kubeConfigFile: /dev/null
+        kind: WebhookAdmission
+    ValidatingAdmissionWebhook:
+      configuration:
+        apiVersion: apiserver.config.k8s.io/v1alpha1
+        kubeConfigFile: /dev/null
+        kind: WebhookAdmission
+EOT
+
+  oc ex config patch /etc/origin/master/master-config.yaml.prepatch -p "$(cat /etc/origin/master/master-config.patch)" > /etc/origin/master/master-config.yaml
+  /usr/local/bin/master-restart api && /usr/local/bin/master-restart controllers
+
+  rm /etc/origin/master/master-config.yaml.prepatch /etc/origin/master/master-config.patch
+  
+EOF
+
+   timeout_positive 60 "oc login -u system:admin"
```

Comment:
Instead of creating a mostly duplicate function that differs by only one character, I think I'd prefer to just do what the minishift script does, e.g. `until oc login -u system:admin 2>/dev/null; do sleep 5; done;`